platform/android/MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/testapp/maps/widgets/CompassViewTest.java

```diff
@@ -40,19 +38,21 @@ public void testDefault() {
   }
 
   @Test
+  @Ignore
   public void testVisible() {
```

Comment:
We should remove `test` prefix from tests names. It's implicit with the usage of `@Test` annotation. What about `checksVisibility`?